### PR TITLE
Show

### PR DESCRIPTION
**Pull request by ValentinGaudin**

### CODE REVIEW

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
Conflicts with dev

**Comment by ValentinGaudin** [comment]
Conflicts have been resolve.

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
No we don't want your ugly macOs files (public/assets/images/.DS_Store)

File: src/View/Offering/search.html.twig [hunk 1]
```diff
@@ -46,13 +46,21 @@
 						{% for offer in offers %}
 
 							<div class="offer bg-light border border-muted mx-auto p-4 mb-4 text-left col-12 col-lg-5">
+<<<<<<< HEAD
+								<a href="/offerings/show?id={{offer.id}}" class="offer-title-link text-decoration-none">
+=======
 								<a href="/offerings/show?id={{offer.id}}&userid={{offer.userid}}" class="offer-title-link text-decoration-none">
+>>>>>>> f71fc3331b7e2b63d59fa131865f29d688cbb494
```
Comment (aldaitzwild) [inline src/View/Offering/search.html.twig:53]: Youhou ! you send a conflict !

File: src/View/Offering/search.html.twig [hunk 2]
```diff
@@ -74,4 +82,8 @@
 
 	</div>
 
+<<<<<<< HEAD
 {% endblock %}
+=======
+{% endblock %}
+>>>>>>> f71fc3331b7e2b63d59fa131865f29d688cbb494
```
Comment (aldaitzwild) [inline src/View/Offering/search.html.twig:89]: And another one !!!

File: src/View/layout.html.twig [hunk 3]
```diff
@@ -12,6 +12,10 @@
     <link rel="stylesheet" href="/assets/css/_style.css">
     <link rel="stylesheet" href="/assets/css/style.css">
     <link rel="icon" href="/assets/images/favicon.png">
+<<<<<<< HEAD
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+</head>
+=======
```
Comment (aldaitzwild) [inline src/View/layout.html.twig:18]: Never 3 without 4 right ?

File: src/View/layout.html.twig [hunk 4]
```diff
@@ -82,7 +87,10 @@
             {% block javascript %}
            
             {% endblock %}
+<<<<<<< HEAD
+=======
 
+>>>>>>> f71fc3331b7e2b63d59fa131865f29d688cbb494
```
Comment (aldaitzwild) [inline src/View/layout.html.twig:93]: A fifth ? Ok ...

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Review by ValentinGaudin — COMMENTED** [review]
I think it's all resolve now. -_-

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
Conflicts with dev

**Comment by ValentinGaudin** [comment]
Resolve the conflicts after another PR.

**Pull request merged**